eslint.config.mjs

```diff
@@ -134,6 +136,31 @@ export default tseslint.config([
       // We violate this rule in a lot of places. We'll turn it off for now.
       'vitest/no-identical-title': ['off'],
 
+      // Use the recommended rules for HTML.
+      ...Object.fromEntries(
+        Object.keys(html.rules).map((value) => ['@html-eslint/' + value, 'error']),
+      ),
+      // We don't want these style rules
+      '@html-eslint/attrs-newline': 'off',
+      '@html-eslint/element-newline': 'off',
+      '@html-eslint/indent': 'off',
+      '@html-eslint/no-inline-styles': 'off',
+      '@html-eslint/no-trailing-spaces': 'off',
+      '@html-eslint/sort-attrs': 'off',
+      // We don't want these rules
+      '@html-eslint/no-heading-inside-button': 'off', // not important
+      '@html-eslint/require-explicit-size': 'off', // we don't always have sizes when we use classes.
+      '@html-eslint/require-form-method': 'off', // default is 'GET', that's fine.
+      '@html-eslint/require-input-label': 'off', // we don't always have labels.
+      // We prefer tags like `<img />` over `<img>`.
+      '@html-eslint/no-extra-spacing-attrs': ['error', { enforceBeforeSelfClose: true }],
+      '@html-eslint/require-closing-tags': ['error', { selfClosing: 'always' }],
+      // False positives for attribute/element baseline browser compatibility.
+      '@html-eslint/use-baseline': 'off',
+      // We violate these rules in a lot of places.
+      '@html-eslint/id-naming-convention': 'off',
+      '@html-eslint/require-button-type': 'off',
```

Comment:
Should we enable this `@html-eslint/require-button-type` rule?

Reply:
I wouldn't mind this, we've had issues in the past with buttons that were inadvertently set to submit (see #10388).

Reply:
See #12372